guides/v2.2/frontend-dev-guide/theme-best-practice.md

```diff
@@ -7,26 +7,22 @@ functional_areas:
   - Standards
 ---
 
-This topic describes the recommended approaches and things to consider when working on a custom theme.
+Utilizing theme development best practices give you a better chance of avoiding conflicts and issues with your theme after you update or upgrade your Magento instance or install a custom extension.
```

Comment:
maybe add "customized" before theme.  

also, i see this a lot, but aren't update and upgrade the same thing?

Reply:
Update and upgrade are not always the same thing, and definitely not always the same for Magento. You can upgrade from Open Source to Commerce, but you can't update from there to there. You can update your installation with hotfixes, but that wouldn't be considered an upgrade.

Reply:
valid.  wasn't thinking from edition to edition, was thinking version updates.